test/ampycloud/plots/test_core.py

```diff
@@ -42,16 +43,50 @@ def test_large_dts(mpls):
     # Run ampycloud
     chunk = run(data)
 
-    # Uncomment the line below once pytst 7.0 can be pip-installed
-    #with pytest.does_not_warn():
-    if True:
+    # The following should not trigger any warning... let's make sure of that.
+    with warnings.catch_warnings():
+        warnings.simplefilter("error")
         diagnostic(chunk, upto='layers', show_ceilos=False, show=False,
                    save_stem='pytest_large_dts', save_fmts='png',
                    ref_metar_origin='Large dts', ref_metar='OVC010')
 
     assert Path('pytest_large_dts.pdf').exists
 
 
+def test_direct_prms(mpls):
+    """ Test that all goes well if users feed params directly to run().
+
+    Args:
+        mpls: False, or the value of MPL_STYLE requested by the user. This is set automatically
+            by a fixture that fetches the corresponding command line argument.
+            See conftest.py for details.
+
+    """
+
+    if mpls:
+        dynamic.AMPYCLOUD_PRMS['MPL_STYLE'] = mpls
+
+    # Let's create some data with only NaNs
```

Comment:
why only nans?

Reply:
This test is to make sure we can create empty plots, i.e. plots when the sky is clear with 0 clouds.